src/platform/silabs/BLEManagerImpl.h

```diff
@@ -208,6 +208,11 @@ class BLEManagerImpl final : public BLEManager, private BleLayer, private BlePla
     static void DriveBLEState(intptr_t arg);
     static void BleAdvTimeoutHandler(TimerHandle_t xTimer);
     uint8_t GetTimerHandle(uint8_t connectionHandle, bool allocate);
+
+#if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+    uint16_t Internal_Slow_Advertising_MIN = CHIP_DEVICE_CONFIG_BLE_SLOW_ADVERTISING_INTERVAL_MIN;
+    uint16_t Internal_Slow_Advertising_MAX = CHIP_DEVICE_CONFIG_BLE_SLOW_ADVERTISING_INTERVAL_MAX;
+#endif
```

Comment:
This does not need to be in the header. Recommend only having this in the impl file (e.g. BLEManagerImpl)

Reply:
member variable names use `mCamelCase`. Please do not use SnakeCase and use `Min/Max` without uppercasing them.

Reply:
@andy31415 Noted for next time, for this one I will step away from the member to using the constants directly as requested above.